src/main/kotlin/org/onflow/flow/sdk/models.kt

```diff
@@ -675,6 +675,18 @@ data class FlowBlock(
     }
 }
 
+data class ExecutionResult(
+    val id: FlowId,
+    val parentId: FlowId
+) : Serializable {
+    companion object {
+        fun of(grpcExecutionResult: Access.ExecutionResultByIDResponse) = ExecutionResult(
+            id = FlowId.of(grpcExecutionResult.executionResult.blockId.toByteArray()),
+            parentId = FlowId.of(grpcExecutionResult.executionResult.previousResultId.toByteArray())
```

Comment:
```suggestion
            blockId = FlowId.of(grpcExecutionResult.executionResult.blockId.toByteArray()),
            previousResultId = FlowId.of(grpcExecutionResult.executionResult.previousResultId.toByteArray())
```

Reply:
@jribbink Noted, thanks! Just pushed the following updates: 
- renamed `id` to `blockId`
- renamed `parentId` to `previousResultId`
- added chunks and serviceEvents to the FlowExecutionResult class
- added classes for FlowChunk and FlowServiceEvent

Thank you!

Reply:
Awesome, will take a look by EOD